Undergraduate student projects in the lab
=============

* We want to cultivate a lab environment so that undergraduates feel like part of our research group.
* Pick project that you're interested in.
* What time can you commit? 
* What do you want to get out of it?


Projects:

1. **Testing pipelines:** Data analysis pipelines need testing to ensure they are reproducible. We will supply raw data and scripts. You will try getting the scripts to run on your own computer and take detailed notes when something does not work. Troubleshooting will be necessary because inevitably software will break, there might be a typo or something we missed while rushing through an analysis. **Note:** We will expect you to take on a level of independence and initiative with asking questions when something doesn't make sense. It's not your fault! Clarity with instructions is probably due to missing information in the code. Send us updates with what you're interested in. Engage in discussion surrounding the analysis. See what you can find! Perhaps it is different than our results, and that would be cool! There is potential to gain ideas for your own project to stem off of these analyses. Examples of re-analysis projects available: 
    * [Jane's  transcriptome data](https://github.com/janejpark/niehs)
    * [Lisa's de novo assembly/evaluation/annotation scripts](https://github.com/ljcohen/osmotic) 
    * [Lisa's gene expression analyses](https://github.com/ljcohen/RNAseq_15killifish)
    * Others?

2. **[Bioinformatics programming](https://github.com/WhiteheadLab/Lab_Wiki/blob/master/bioinformatics_programming.md):** Work with Lisa on solving problems and going through tutorials related to bioinformatics. Some of these exercises are short. They can be completed on your own computer when there is free time from other projects. Some level of independence and initiative is necessary to work through these exercises on your own, send updates, ask questions/engage in discussion. Ask questions!

3. **Abalone Comparative Physiology** - Research Questions:
    A. Does parental diet conditioning affect offspring during senstive developmental time points? Does parental diet buffer against ocean acidification conditions during sensitive time points? 80 broodstock  sourced from The Cultured Abalone Farm (Goleta, CA) were sexed, gonads scored and split into two diet treatments in December 2017 - red algae (Dulse - high protein content) and brown algae (Giant Kelp - high carbohdrate content. We hypothesize offspring generated from broodstock fed high protein content will be more tolerant to low pH conditions due to higher quality nutrient partitioning during gametogenesis. 
    B. Seawater buffering - during the sensitive planktonic larval phase, offspring generated from  red/brown algae spawn will be cultured under ambient and low pH conditions during the free-swimming, lecithotrophic (non-feeding) phase for 7 days. Survival will be assessed 4 days post-hatch (dph) & 7dph. Samples for total lipid analysis will be photographed and collected for processing at a later date.  
    C.Modulate temperature & pH during juvenile grow-out phase to increase growth rates. Settlement will be induced for the planktonic larvae using GABA 7dph. At this time, 
    D. Use specific parental lines to generate tolerant offspring to increase red abalone survival and performance under acidified conditions.
    E. Potenital projects acute stressor exposure, behavior trials, etc.


4. **Abalone morphometrics data** -  Size analysis using imageJ. Red (*Haliotis rufescens*) and white (*Haliotis sorenseni*) abalone were spawned under ambient and experimental pH during early development, a critical time point sensitive to environmental perturbations. Abalone was assessed at 4 days post-hatch (dph), 7 dph, and 10 dph, which corresponds to 3 days post-settlement induction. At these timepoints, developing abalone were photographed and archived for total lipid analysis. 
    A. Potential Project - Assist in size analyses for various abalone experiments.


5. **K. mar dose-response** [Elias]
* Will involve dealing with live fish and experimenting with them and their offspring:
    * Raising fish in lineages
    * Collecting offspring and dosing with environmentally relevant compounds
    * Measuring physiological response to those compounds (qPCR; microscopy; fluorescent microscopy)

6. **Pacific herring genome annotations** [Elias]
    * Involves working with me to use existing data (transcriptome and orthology based) to annotate the genome
    * will spend time reading about annotation
    * I'll provide coding exercises and you will work with me on code and proofreading/testing code

7. **SETAC, norCal proposal**
This would be a good avenue for a small pot of funding for a small dose response type experiment if an undergrad has  a good idea. $1000 and a high rate of funding; recipient must present at the NORCAL SETAC. Really good for students interested in agency and regulatory. Great for an academic CV as well.

8. **Pacific herring qPCR gene expression** [Tony]
qPCR assay to quantify expression of MX1 (viral response marker gene) from virus challenge study--4 immunocompetent tissues across 12 timepoints during infection. MX1 expression is routinely used to validate whether fish were exposed to live virus. The idea will be to look for differential expression between treatments to determine onset and response to infection. Project will involve learning RNA extraction, primer design, qPCR troubleshooting followed by statistical analsyis in R or Excel. **1 person project** 

9. **Hi-C pipeline development** [Tony]
[apply cutting-edge methods for hybrid genome assembly](https://www.nature.com/articles/nrm.2016.104) For someone interested in a challenging bioinformatics project. You will work closely with me to troubleshoot and implement a computational pipeline for illumina sequence data from a Hi-C library. You will spend time reading about Hi-C analysis. **1 person project**

10. **SNP calling from tag-seq data** [Tony & Jane]
herring project: both oil exposed and control fish came from the same sets of parents so expectation is that both groups share the same genotypes throughout our experiments. We observed a variance in response between two treatment groups: oil + virus and no oil + virus. Your project is to mine SNPs from RNA-seq data to test the hypothesis that diferential reponse to virus challenge between these two groups is result of oil exposure favoring some genotypes over others. This is a computational project analyzing a subset of existing data. **1 person project**

11. **RNA-SEQ pipeline comparisons** [Jane]
Run through complete differential expression analyses to compare results from established R packages and the custom analysis pipeline Jane developed. Super important for validating outcomes that Jane identified so far, with potential to be included as part of a paper or supplemental materials. 
